x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_setup_flyout.tsx

```diff
@@ -6,31 +6,31 @@
  */
 
 import React from 'react';
-import { CoreStart } from '@kbn/core/public';
-import { lastValueFrom } from 'rxjs';
+import type { CoreStart } from '@kbn/core/public';
 import { toMountPoint, wrapWithTheme } from '@kbn/kibana-react-plugin/public';
+import { extractInfluencers } from '../../../common/util/job_utils';
 import { VIEW_BY_JOB_LABEL } from '../../application/explorer/explorer_constants';
 import { AnomalySwimlaneInitializer } from './anomaly_swimlane_initializer';
-import { AnomalyDetectorService } from '../../application/services/anomaly_detector_service';
 import { getDefaultSwimlanePanelTitle } from './anomaly_swimlane_embeddable';
 import { HttpService } from '../../application/services/http_service';
-import { AnomalySwimlaneEmbeddableInput } from '..';
+import type { AnomalySwimlaneEmbeddableInput } from '..';
 import { resolveJobSelection } from '../common/resolve_job_selection';
+import { mlApiServicesProvider } from '../../application/services/ml_api_service';
 
 export async function resolveAnomalySwimlaneUserInput(
   coreStart: CoreStart,
   input?: AnomalySwimlaneEmbeddableInput
 ): Promise<Partial<AnomalySwimlaneEmbeddableInput>> {
   const { http, overlays } = coreStart;
 
-  const anomalyDetectorService = new AnomalyDetectorService(new HttpService(http));
+  const { getJobs } = mlApiServicesProvider(new HttpService(http));
 
   return new Promise(async (resolve, reject) => {
     try {
       const { jobIds } = await resolveJobSelection(coreStart, input?.jobIds);
       const title = input?.title ?? getDefaultSwimlanePanelTitle(jobIds);
-      const jobs = await lastValueFrom(anomalyDetectorService.getJobs$(jobIds));
-      const influencers = anomalyDetectorService.extractInfluencers(jobs);
+      const { jobs } = await getJobs({ jobId: jobIds.join(',') });
```

Comment:
I've switched this to use our standard `getJobs` function as it doesn't need to use the observable based one supplied by the `AnomalyDetectorService` class.